From 7e7fa5f8719c0a497f4b262e6fb5625c13b6c22e Mon Sep 17 00:00:00 2001
From: David S. Miller <davem@davemloft.net>
Date: Fri, 16 Mar 2012 20:40:54 -0700
Subject: [PATCH] Fix libc-lowlevellock.c sysdep finding more generically.

nptl/

	[BZ #13844]
	* sysdeps/unix/sysv/linux/libc-lowlevellock.c: Include using <..>
	instead of "...".
	* sysdeps/unix/sysv/linux/sparc/sparc32/libc-lowlevellock.c:
	Delete, not needed.

Index: b/nptl/sysdeps/unix/sysv/linux/libc-lowlevellock.c
===================================================================
--- a/nptl/sysdeps/unix/sysv/linux/libc-lowlevellock.c
+++ b/nptl/sysdeps/unix/sysv/linux/libc-lowlevellock.c
@@ -18,4 +18,4 @@
    02111-1307 USA.  */
 
 /* No difference to lowlevellock.c, except we lose a couple of functions.  */
-#include "lowlevellock.c"
+#include <lowlevellock.c>
